kahuna/public/js/upload/jobs/upload-jobs.js

```diff
@@ -1,178 +1,196 @@
                 // archived, etc), refresh the copy we hold
-                $rootScope.$on('image-updated', (e, updatedImage) => {
-                    if (updatedImage.data.id === image.data.id) {
-                        jobItem.image = updatedImage;
-                    }
+                $rootScope.$on("image-updated", (e, updatedImage) => {
+                  if (updatedImage.data.id === image.data.id) {
+                    jobItem.image = updatedImage;
+                  }
                 });
 
                 // we use the filename of the image if the description is missing
                 if (!jobItem.image.data.metadata.description) {
-                    const newDescription = jobItem.name
-                        .substr(0, jobItem.name.lastIndexOf('.'))
-                        .replace(/_/g, ' ');
-
-                    editsService.updateMetadataField(jobItem.image, 'description', newDescription);
+                  const newDescription = jobItem.name
+                    .substr(0, jobItem.name.lastIndexOf("."))
+                    .replace(/_/g, " ");
+
+                  editsService.updateMetadataField(
+                    jobItem.image,
+                    "description",
+                    newDescription
+                  );
                 }
 
                 if (presetLabels.length > 0) {
-                    labelService.add(image, presetLabels);
+                  labelService.add(image, presetLabels);
                 }
 
                 $rootScope.$emit(
-                  'track:event',
+                  "track:event",
                   eventName,
                   null,
-                  'Success',
+                  "Success",
                   null,
-                  { 'Labels' : presetLabels.length}
+                  { Labels: presetLabels.length }
                 );
-            }, error => {
-                jobItem.status = 'upload error';
+              },
+              error => {
+                jobItem.status = "upload error";
                 jobItem.error = error.message;
 
                 $rootScope.$emit(
-                  'track:event',
+                  "track:event",
                   eventName,
                   null,
-                  'Failure',
+                  "Failure",
                   null,
-                  { 'Failed on': 'index'}
+                  { "Failed on": "index" }
                 );
-            });
-        }, error => {
+              }
+            );
+          },
+          error => {
             const reason = error.body && error.body.errorKey;
 
-            const message = reason === 'unsupported-type' ?
-                'The Grid only supports JPG and PNG images.' +
-                ' Please convert the image and try again.' :
-                error.body && error.body.errorMessage || 'unknown';
+            const message =
+              reason === "unsupported-type"
+                ? "The Grid only supports JPG and PNG images." +
```

Comment:
Unrelated but does this message need changing now TIFF support has been added? cc @paperboyo

Reply:
Thanks for review, @mbarton! Yeah, also seen it, but TIFF support hasn’t been advertised yet. Reasoning (not too strong, I admit) being outstanding [work in this](https://github.com/guardian/grid/issues/2395).